.jenkins/Jenkinsfile-cryptography-wheel-builder

```diff
@@ -16,22 +16,20 @@ def configs = [
     ],
     [
         label: 'sierra',
-        versions: ['py27', 'py34', 'py35', 'py36'],
+        versions: ['py27', 'py34'],
```

Comment:
Can you add comments indicating that whatever the py3 version listed here is the minimum ABI version for the wheels?

Reply:
(Same with all the other sites as well)